src/fiatExchanges/FiatExchangeCurrency.tsx

```diff
@@ -142,15 +144,27 @@ function FiatExchangeCurrency({ route, navigation }: Props) {
           />
           <CurrencyRadioItem
             title="CELO"
-            selected={selectedCurrency === Currency.Celo}
-            onSelect={() => setSelectedCurrency(Currency.Celo)}
+            selected={selectedCurrency === CiCoCurrency.CELO}
+            onSelect={() => setSelectedCurrency(CiCoCurrency.CELO)}
             enabled={flow !== FiatExchangeFlow.Spend}
+            containerStyle={{
+              borderTopWidth: 0.5,
+              borderBottomWidth: 0.5,
+            }}
```

Comment:
nit: now that there are multiple middle currency radio items, the style could be re-used between them